src/pages/mathematics/intro-to-inverse-functions/index.md

```diff
@@ -2,8 +2,14 @@
 title: Intro to Inverse Functions
 ---
 ## Intro to Inverse Functions
+  <p>Say you were trying to reflect y=3xover the line y=x. What would be the equation of the line? As you can see in the image below, it would be y=1/3x. But why is that so? If you swap x and y with each other and solve for y, you can find that the equation would be the same. This is what would be called a inverse function.
+<img src = "http://d2r5da613aq50s.cloudfront.net/wp-content/uploads/369430.image1.jpg" />
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/intro-to-inverse-functions/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+
+###Sources:
+  “How to Graph the Inverse of a Function.” Dummies, www.dummies.com/education/math/calculus/how-to-graph-the-inverse-of-a-function/.
```

Comment:
This link isn't formatted properly. Check [here](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#links) for guidance on how to do this in Markdown.